protocols/servlet-jakarta/src/main/java/org/jboss/arquillian/protocol/servlet5/v_5/ServletProtocolDeploymentPackager.java

```diff
@@ -92,7 +93,7 @@
     private Archive<?> handleArchive(JavaArchive applicationArchive, Collection<Archive<?>> auxiliaryArchives,
         JavaArchive protocol, Processor processor) {
         return handleArchive(
-            ShrinkWrap.create(WebArchive.class, "test.war")
+            ShrinkWrap.create(WebArchive.class, UUID.randomUUID().toString() + ".war")
```

Comment:
```suggestion
            ShrinkWrap.create(WebArchive.class, "test-" + UUID.randomUUID().toString() + ".war")
```

I would keep `test-` prefix at least. This way it's more obvious what this archive could be.